javascript/javascript2/week1/nameGenerator.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<body>  
+   userName:tariq <br>
+ <input type="text" id="myText" onclick="empty()" >
```

Comment:
You are mixing `onclick` behaviour in HTML and in javascript, this is very bad practice. just put the `empty` function inside
```
document.getElementById("myBtn").addEventListener("click", function() {
...
}
```